CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+# Contributing
+
+From opening a bug report to creating a pull request: every contribution is
+appreciated and welcome. If you're planning to implement a new section or page 
+please create an issue first. This way we can ensure that your precious
+work is not in vain.
+
+There are two types of contributions for the wiki:
+
+* **Content Changes**  
+  *Additions and changes to the content of the Wiki.*  
+
+* **Code Changes**  
+  *Enhancements and fixes to the code that runs the Wiki.*
+
+We greatly appreciate contributions to either one!
+
+## Setup
+
+* Install [Node.js](https://nodejs.org/) if you have not already
```

Comment:
I'd mention that atleast node 6 may be required for "best results". I had heard from a few people that they were having issues with rendering on node ~v4